src/InventoryList/Storage.pm

```diff
@@ -0,0 +1,59 @@
+package InventoryList::Storage;
+
+use strict;
+use Globals;
+use InventoryList;
+use base qw(InventoryList);
+
+sub new {
+	my ($class) = @_;
+	my $self = $class->SUPER::new;
+	$self->{openedThisSession} = 0;
+	$self->{opened} = 0;
+	$self->{items} = 0;
+	$self->{items_max} = 0;
+	return $self;
+}
+
+sub isOpenedThisSession {
+	my ($self) = @_;
+	return $self->{openedThisSession} == 1;
+}
+
+sub isOpened {
```

Comment:
Maybe name it `isReady`, as in Inventory and Cart if they all have the same purpose?